docs/start.rst

```diff
@@ -78,6 +78,38 @@ run the commands below.
         --start-date 2015-01-01 \
         --end-date 2015-01-02
 
+Basic Airflow architecture
```

Comment:
What do you think about moving this to a separate page? This is useful not only for people who want to start using, but for anyone who wants to understand how it works.

Reply:
Any suggestions where do you think would we should keep it?

A main reason to have it over was the new users know the basic architecture but we are open for suggestions

Reply:
I think we can put it between UI and Concepts
<img width="256" alt="Screenshot 2020-08-20 at 22 05 32" src="https://user-images.githubusercontent.com/12058428/90820227-5d832580-e331-11ea-9126-fd481ebb8a6c.png">

Reply:
But it can stay here as well. I don't have a strong opinion. It won't be a problem to move it in the future when we clean up some docs.  I am happy that you wanted to describe it to us in the documentation. It's very helpful.

Reply:
Cool, yeah let's keep it as it is for now. As we add more we can move everything to a dedicated section :)

Reply:
@potiuk Looks like you mentioned entire Apache committers again, can we stop mentioning any list at all and just mention people by their ids, thus avoiding this issues please.